http-server/src/test/java/org/triplea/server/lobby/chat/MessageBroadcasterTest.java

```diff
@@ -0,0 +1,42 @@
+package org.triplea.server.lobby.chat;
+
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.google.common.collect.Sets;
+import java.util.function.BiConsumer;
+import javax.websocket.Session;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.triplea.http.client.lobby.chat.events.server.ServerMessageEnvelope;
+
+@ExtendWith(MockitoExtension.class)
+class MessageBroadcasterTest {
+
+  @Mock private Session session0;
+  @Mock private Session session1;
+  @Mock private Session session2;
+  @Mock private ServerMessageEnvelope serverEventEnvelope;
+
+  @Mock private BiConsumer<Session, ServerMessageEnvelope> singleMessageSender;
+  @InjectMocks private MessageBroadcaster messageBroadcaster;
+
+  @Test
+  void accept() {
+    when(session0.getOpenSessions()).thenReturn(Sets.newHashSet(session0, session1, session2));
```

Comment:
Actually java has now those kind of factory methods as well, see https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Set.html#of(E,E,E)

Reply:
Thanks, I was having trouble finding that method, updated: 9d7259218